### ISSUE TITLE
Make SteamLauncher work when Kodi is in standalone

### ISSUE THREAD

**Issue by chris-33** [teeedubb/teeedubb-xbmc-repo#11]
Do you have any ideas on how to make the SteamLauncher work when Kodi is in standalone mode? I know it's missing the window manager, but that seems like it shouldn't be hard to fix. I've tried starting openbox before the steam executable is run, but that doesn't work.

_Aside:_ What I've actually done is ssh into my HTPC while Kodi is running in standalone, then run `openbox --replace --sm-disable --startup "steam -bigpicture"`. It seemed like if I can get it running like that, it should be fairly easy to turn this into a solution using the pre-run hook of your addon.

I know steam is started, because I get the intro sounds and the GUI sounds. But I don't see anything, other than the "updating steam" message.

Any ideas?

**Comment by teeedubb** [comment]
I could never get steam bpm to work with out a window manager. Non bpm steam is ok though. If you work it out please share.

**Comment by chris-33** [comment]
No, I couldn't get it to work either. Like I said, I know it was running because I could hear the sounds, but no dice on the graphics. Seems like in theory starting Openbox before launching Steam should be enough, but oh well. I'm now just launching Kodi on top of an Openbox session and that's much easier.

Side note if you're interested: I tried [this approch](http://ubuntuforums.org/showthread.php?t=2261037) more successfully - basically it uses dm-tool to switch between users, each set to autologin to a dedicated session. Ultimately I gave that up as well though, it was unreliable on the switch back to Kodi and I was worried the hard kill might leave Kodi's database corrupted eventually. Just figured you might be interested. ;)
